### DISCUSSION TITLE
Composer 1 & 2 cache mixing

### DISCUSSION THREAD

**Discussion by carlos-reynosa** [markshust/docker-magento#687] [Q&A]
What do you think would be the effects on your local install if you have composer 1 installed locally but you run this project and it installs composer 2 packages within your composer 1 cache?

https://github.com/markshust/docker-magento/blob/18bc135927cbc401b7bec56c5433506c3c92f1eb/compose/docker-compose.yml#L22

**Comment by rangerz** [comment]
I'm not sure what the effects will be.

Try to clean all cache to avoid it

```
composer clearcache

# even clean manual
rm -rf ~/.composer/cache
```

Or you should prepare the different folders for composer v1 and v2
